### PR TITLE
Deployment structure and documentation

### PR DESCRIPTION
**Pull request by rzadp**
First step in preparation for #9, #10, #11 

- Merged `deployment/oracle/README.md` and `deployment/README.md`
- Moved `deployment/oracle/*` to `deployment/*`
- Abstracted out Configuration and Execution as the readme got very long

### CODE REVIEW

File: deployment/CONFIGURATION.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  * Record the IP address (required for file setup).
+  * Setup ssh access to your node via public+private keys (using passwords is less secure). 
+  * When creating the node, set a meaningful `hostname` that can identify you (e.g. `validator-0x...`).
+
+## Initialization
+
+1. Clone this repository and go to the `deployment/oracle` folder
+```
+git clone --recursive https://github.com/poanetwork/tokenbridge
+cd tokenbridge/deployment
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: cd tokenbridge/deployment/oracle

File: deployment/CONFIGURATION.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Prerequisites
+
+A functional Ubuntu 16.04 server launched using a trusted hosting provider. For more information, see our tutorials on [setting up a validator node on AWS](https://github.com/poanetwork/wiki/wiki/Validator-Node-on-AWS) or [setting up on non-AWS](https://github.com/poanetwork/wiki/wiki/Validator-Node-Non-AWS).
+  * Record the IP address (required for file setup).
+  * Setup ssh access to your node via public+private keys (using passwords is less secure). 
+  * When creating the node, set a meaningful `hostname` that can identify you (e.g. `validator-0x...`).
+
+## Initialization
+
+1. Clone this repository and go to the `deployment/oracle` folder
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: go to the `deployment` folder

File: deployment/CONFIGURATION.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Administrator Configurations
+
+1. The `group_vars/<bridge_name>.yml` file contains the public bridge parameters. This file is prepared by administrators for each bridge. The validator only needs to add the required bridge name in the hosts.yml file to tell Ansible which file to use.
+
+   `group_vars/example.yml` shows an example configuration for the POA/Sokol - POA/Sokol bridge. Parameter values should match values from the .env file for the Oracle. See [Configuration parameters](../../oracle/README.md#configuration-parameters) for details.
+
+2. You can also add the following parameters in the `group_vars` to change the default behavior of the playbooks:
+
+2.1 `compose_service_user` - specifies users to be created by playbooks. This user will be used to run Token Bridge Oracle.
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: This user runs the Token Bridge Oracle

File: deployment/CONFIGURATION.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+2.2 `bridge_repo` contains address of Token Bridge Oracle repository. The default value is https://github.com/poanetwork/tokenbridge.
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: contains the address of the Token Bridge Oracle repository. The default value is...

File: deployment/CONFIGURATION.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+2.3 `bridge_repo_branch` points to the specific branch or commit to use with the `bridge_repo`. If `bridge_repo_branch` is not specified, the default (`master`) branch is used.
+
+2.4 `bridge_path` set the path where Token Bridge Oracle would be installed. By default it point to the home folder of `compose_service_user`
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: sets the path where the Token Bridge Oracle is installed. By default, it points...

File: deployment/CONFIGURATION.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
+# POA Token Bridge / Deployment Configuration
```
Comment (andogro) [inline deployment/CONFIGURATION.md]: We should be consistent throughout the repo, and rename Token Bridge / Token bridge to TokenBridge.

File: deployment/CONFIGURATION.md (hunk 2 above)
Comment (rzadp) [reply deployment/CONFIGURATION.md]: Updated.

File: deployment/CONFIGURATION.md (hunk 3 above)
Comment (rzadp) [reply deployment/CONFIGURATION.md]: Yes, it's a single `poadocker` user. Updated.

File: deployment/CONFIGURATION.md (hunk 4 above)
Comment (rzadp) [reply deployment/CONFIGURATION.md]: Updated.

File: deployment/CONFIGURATION.md (hunk 5 above)
Comment (rzadp) [reply deployment/CONFIGURATION.md]: Updated.

**Comment by rzadp** [comment]
@andogro Thanks for the review. In comments like these, where you suggest exact words to be used, you might want to use the [Suggesting Changes](https://haacked.com/archive/2019/06/03/suggested-changes/) feature. It looks more clear plus you'd be counted as a co-commiter.

@akolotov @patitonar @phahulin @ArseniiPetrovich Do you have any thoughts on the PR?

**Comment by akolotov** [comment]
Sorry for delay with review. The changes are OK for me. I think we can merge this PR.

**Pull request merged**